application/src/main/java/org/opentripplanner/service/vehiclerental/model/VehicleRentalStationBuilder.java

```diff
@@ -0,0 +1,364 @@
+package org.opentripplanner.service.vehiclerental.model;
+
+import java.time.Instant;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Consumer;
+import javax.annotation.Nullable;
+import org.locationtech.jts.geom.Geometry;
+import org.opentripplanner.framework.i18n.I18NString;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+
+public class VehicleRentalStationBuilder {
+
+  private final VehicleRentalStation original;
+  private FeedScopedId id;
+  private I18NString name;
+  private String shortName;
+  private Double longitude;
+  private Double latitude;
+  private String address;
+  private String crossStreet;
+  private String regionId;
+  private String postCode;
+  private Set<String> rentalMethods = Set.of();
+  private Boolean isVirtualStation;
+  private Geometry stationArea;
+  private Integer capacity;
+  private Map<RentalVehicleType, Integer> vehicleTypeAreaCapacity = Map.of();
+  private Map<RentalVehicleType, Integer> vehicleTypeDockCapacity = Map.of();
+  private Boolean isValetStation;
+  private VehicleRentalSystem system;
+  private VehicleRentalStationUris rentalUris;
+  private Integer vehiclesAvailable;
+  private Integer vehiclesDisabled;
+  private Map<RentalVehicleType, Integer> vehicleTypesAvailable = Map.of();
+  private Integer spacesAvailable;
+  private Integer spacesDisabled;
+  private Map<RentalVehicleType, Integer> vehicleSpacesAvailable = Map.of();
+  private Boolean isInstalled;
+  private Boolean isRenting;
+  private Boolean isReturning;
+  private Instant lastReported;
+  private Boolean overloadingAllowed;
+  private Boolean isArrivingInRentalVehicleAtDestinationAllowed;
+  private Boolean realTimeData;
+
+  VehicleRentalStationBuilder(VehicleRentalStation original) {
+    this.original = original;
+  }
+
+  public VehicleRentalStation original() {
+    return original;
```

Comment:
This uses a completely different pattern to the rest of the builders but using `original` in the getters. Can you harmonise it by setting the variables in the constructor?